AK/String.cpp

```diff
@@ -484,6 +484,11 @@ ErrorOr<String> String::trim(StringView code_points_to_trim, TrimMode mode) cons
     return trim(Utf8View { code_points_to_trim }, mode);
 }
 
+ErrorOr<String> String::trim_whitespace(TrimMode mode) const
+{
+    return trim(" \n\t\v\f\r"sv, mode);
```

Comment:
Let's not repeat mistakes from the old string type, "whitespace" has no unique definition. At the very least call it `trim_ascii_whitespace`.